### DISCUSSION TITLE
When will a release be created to fix a High Severity Vulnerability?

### DISCUSSION THREAD

**Discussion by astudd** [nodejs/help#4604] [Q&A]
CVE ID: https://github.com/advisories/GHSA-3xgq-45jj-v275
Severity: High
Description: The vulnerability allows an attacker to exploit an insecure configuration or flaw in the container to gain unauthorized access, escalate privileges, or execute arbitrary code remotely.

This high severity vulnerability has been resolved in NPM and the fix to bump the version was merged 3 days ago (https://github.com/nodejs/node/pull/55951) but there still isn't a new release with this fix so the Docker images still have this vulnerability present, which is blocking my team from releasing new code.  Is there an ETA on the next release (18.20.6)?

**Comment by richardlau** [comment]
No ETA, due to https://github.com/nodejs/node/pull/55951#issuecomment-2492474931.

**Comment by richardlau** [comment]
This is being discussed in https://github.com/nodejs/nodejs-dependency-vuln-assessments/issues/193.
